kolibri/core/assets/src/views/core-base.vue

```diff
@@ -110,6 +115,7 @@
       },
     },
     mounted() {
+      this.updateDocumentTitle(this.title);
```

Comment:
right sorry - that's what I meant. (At the time I was also looking at potentially auto-generating the namespaces based on file paths rather than having them hard-coded)